packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -598,62 +324,69 @@ contract FederatedAttestations is
    * @param v The recovery id of the incoming ECDSA signature
    * @param r Output value r of the ECDSA signature
    * @param s Output value s of the ECDSA signature
-   * @dev Throws if sender is not the issuer, account, or an authorized AttestationSigner
    * @dev Throws if an attestation with the same (identifier, issuer, account) already exists
    */
   function registerAttestation(
     bytes32 identifier,
     address issuer,
     address account,
-    uint256 issuedOn,
+    uint64 issuedOn,
     address signer,
     uint8 v,
     bytes32 r,
     bytes32 s
-  ) public isValidUser(issuer, account) {
-    require(
-      isValidAttestation(identifier, issuer, account, issuedOn, signer, v, r, s),
-      "Signature is invalid"
-    );
+  ) external {
+    // TODO allow for updating existing attestation by only updating signer and publishedOn
```

Comment:
and also `issuedOn` I guess?

Reply:
The `issuedOn` field is included in the signature, so updating it requires issuing a new attestation basically

Reply:
Wouldn't that be the case for the `signer` too? Or do you mean creating an entirely separate route for a new signer without an attestation required?

Reply:
On second thought, would probably be easiest to just validate a new signature to update the attestation

Reply:
I think either way validating the signature would be required (maybe I'm missing something)? That said, offering the possibility to update only the relevant fields in storage may be a nice optimization (i.e. not requiring delete + reupload, but instead allowing just a few fields to be overwritten)? (This can be left as a TODO for the future though, doesn't need to be part of this PR)

Reply:
added TODOs :)